src/test/java/dk/dbc/rawrepo/service/RecordCollectionServiceIT.java

```diff
@@ -459,6 +470,85 @@ void fetchRecordCollection_Merged() throws Exception {
         assertThat("collection is missing record", actual.getMissing().get(0), is(new RecordIdDTO("not found", 123456)));
     }
 
+    @Test
+    void getRecordsBulkv2_Merged() throws Exception {
+        final HashMap<String, Object> params = new HashMap<>();
+        params.put("use-parent-agency", "true");
+
+        final RecordIdCollectionDTO recordIdCollectionDTO = new RecordIdCollectionDTO();
+        final List<RecordIdDTO> recordIdDTOList = new ArrayList<>();
+        recordIdDTOList.add(new RecordIdDTO("04628543", 191919));
+        recordIdCollectionDTO.setRecordIds(recordIdDTOList);
+
+        final PathBuilder path = new PathBuilder("/api/v2/records/bulk");
+        final HttpPost httpPost = new HttpPost(httpClient)
+                .withBaseUrl(recordServiceBaseUrl)
+                .withPathElements(path.build())
+                .withData(recordIdCollectionDTO, MediaType.APPLICATION_JSON);
+        for (Map.Entry<String, Object> param : params.entrySet()) {
+            httpPost.withQueryParameter(param.getKey(), param.getValue());
+        }
+        final Response response = httpClient.execute(httpPost);
+        assertThat("Response code", response.getStatus(), is(200));
+
+        String expected = "001 00 *a04628543*b870970*c20200121012638*d19860731*fa\n" +
```

Comment:
Deet er godt nok en sjov post du har gravet frem :-|

Reply:
Det er en eller anden tilfældig post som havde en autoritetspost, og så har jeg fjernet mange af de felter der ikke var brug for.